tests/baselines/reference/tscWatch/emitAndErrorUpdates/incremental/defaultAndD/deepImportChanges/errors-for-.ts-change.js

```diff
@@ -190,9 +190,6 @@ export {};
       ]
     },
     "exportedModulesMap": {
-      "./a.ts": [
```

Comment:
This is better example of change...
This is the test case that shows changes to transitive imports..